README.md

```diff
@@ -44,9 +44,9 @@ Before configuring Synopsys Security Scan into your azure pipeline, note the fol
 
 ## Synopsys Security Scan - Polaris
 
-Synopsys Security Scan Extension available in the Azure DevOps Marketplace is the recommended solution for integrating Polaris into Azure pipeline.
+Synopsys Security Scan Extension available in the Azure DevOps Marketplace is the recommended solution for integrating Polaris into Azure pipeline. The extension will download the Synopsys Bridge CLI, execute a scan, and offers post-scan features such as break-the-build quality gates.
```

Comment:
change offer to offers or make the sentence in present tense. The Extension downloads Synopsys Bridge CLI.  the before synopsys bridge is not needed.

Reply:
we can have since download CLI is also part of our scan process